packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -2985,3 +3141,62 @@ The following response is only a subset of properties for brevity.
     }
 }
 ```
+
+
+## Log Stream Schema object
+
+<ApiLifecycle access="ea" />
+
+
+The Log Stream Schema is defined using [JSON Schema Draft 2020-12](https://json-schema.org/specification.html) with the following properties:
+
+| Property                       | Description                                | DataType                                          | Nullable | Unique | Readonly |
+|:-------------------------------|:-------------------------------------------|:--------------------------------------------------|:---------|:-------|:---------|
+| $id | URI of Log Stream Schema |String |FALSE| TRUE| TRUE|
+|$schema| JSON Schema version identifier| String| FALSE| FALSE| TRUE|
+|title| Name of the Log Streaming integration| String|FALSE| TRUE| TRUE|
+|type| Type of Log Stream Schema property|String containing `string`, `boolean`, `number`, `integer` or `object` |FALSE|FALSE|TRUE|
+|properties|Log Stream Schema properties object| Object |FALSE|TRUE|TRUE|
+|required| Required properties for this Log Stream Schema object|Array of String|FALSE|TRUE|TRUE|
+|oneOf|A non-empty array of valid JSON schemas (see [oneOf description](#log-stream-schema-object-description-details)) |Array|TRUE|FALSE|TRUE|
+|pattern|For `string` Log Stream Schema property type, specify the regular expression used to validate the property (see [Log Stream Schema Property Types and validation](#log-stream-schema-property-types-and-validation)). |String|TRUE|FALSE|TRUE|
+
+In addition to those, Okta extends [JSON Schema Draft 2020-12](https://json-schema.org/specification.html)
+with the following keywords:
+
+| Property                            | Description                                                       | DataType                                                                  | Nullable | Unique | Readonly |
+| :---------------------------------- | :---------------------------------------------------------------- | :------------------------------------------------------------------------ | :------- | :----- | :------- |
+| writeOnce|Determines whether the property can be updated once it has been created|Boolean|FALSE|FALSE|TRUE|
+| errorMessage|Error messages for properties of this Log Stream object|[Error Message object](#error-message-object)|FALSE|TRUE|TRUE|
+
+#### Log Stream Schema object description details
+
+* All Log Stream Schema root object `properties` contain `name`, which specifies the Log Stream name within Okta, and `settings`, which lists properties required to configure Log Stream.
```

Comment:
```suggestion
* All Log Stream Schema root object `properties` contain `name` and `settings`:
     - `name` &mdash; specifies the Log Stream name within Okta
     - `settings` &mdash; lists properties required to configure Log Stream
```
Using to commas to separate entities and their parenthetical descriptions could be confusing. Perhaps a list provides some clarity?